### PR TITLE
Documentation for Iroha installation security tips

### PR DESCRIPTION
**Pull request by Baydarich**
### Description of the Change

Added tips to Read the Docs for securing Iroha installation ranging from physical security to logging. The guide itself is more general and aimed at meeting obvious yet necessary requirements.

### Benefits

The guide helps administrators to avoid possible security problems that are not directly related to Iroha.

### Possible Drawbacks 

None

### CODE REVIEW

**Comment by sudomann** [comment]
@Baydarich I recently created a pull request regarding deploying ansible in kubernetes clusters. I have no experience in security of any sort; could  you make some suggestions as to how i could improve security handling?
Best I have thought of is for the generated files containing private keys to be deleted by ansible after being used to deploy.

File: docs/source/guides/sec-install.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Change any default passwords that are used during installation, e.g., password for connecting to postgres.
+Iroha repository contains examples of private and public keys - never use it in production.
+Moreover, verify that new keypairs are generated in a safe environment and only administrator has access to those keypairs (or at least minimise the number of people).
+After deploying keys to Iroha peers delete private keys from the host that was used to perform deployment, i.e. private keys should reside only inside Iroha peers.
+Create an encrypted backup of private keys before deleting them and limit the access to it.
+
+Network configuration
+^^^^^^^^^^^^^^^^^^^^^
+Iroha listens on ports 50051 and 10001.
+Firewall settings must allow incoming/outcoming connections to/from these ports.
```
Comment (bakhtin) [inline docs/source/guides/sec-install.rst]:
```suggestion
Firewall settings must allow incoming/outgoing connections to/from these ports.
```

File: docs/source/guides/sec-install.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+^^^^^^^
+Install latest operating system security patches and update it regularly.
+If Iroha is running in Docker containers, update Docker regularly.
+While being optional, it is considered a good practice to test updates on a separate server before installing to production.
+
+Logging and monitoring
+^^^^^^^^^^^^^^^^^^^^^^
+- Collect and ship logs to a dedicated machine using an agent (e.g., Filebeat).
+- Collect logs from all Iroha peers in a central point (e.g., Logstash).
+- Enable docker healthcheck.
```
Comment (bakhtin) [inline docs/source/guides/sec-install.rst]: Right now we do not have a mechanism to identify the health of a running container. There are custom implementations that use gRPC to test if Iroha is ready to accept connections (e.g., https://github.com/d3ledger/notary/blob/7b1796472538c33817ebfea67f436221285ebc7d/docker/grpc-healthcheck.dockerfile) but this is not very reliable indicator.

Comment (Baydarich) [reply docs/source/guides/sec-install.rst]: Done - healthcheck mentioning is deleted.

**Comment by Baydarich** [comment]
@sudomann I left a comment in your pull request

**Pull request merged**